docs/visualize/timelion.asciidoc

```diff
@@ -504,3 +504,61 @@ image::images/timelion-conditional04.png[]
 {nbsp}
 
 For additional information on Timelion conditional capabilities, go to https://www.elastic.co/blog/timeseries-if-then-else-with-timelion[I have but one .condition()].
+
+[float]
+[[timelion-deprecation]]
+=== Timelion App deprecation
+
+In *8.0* and later, the Timelion app is going to be deprecated. There are actions you need to complete to copy your existing Timelion worksheets to a dashboard.  
+
+NOTE: Only the Timelion app is deprecated, not the Timelion editor. 
```

Comment:
I am not sure if people will understand "Timelion editor". Maybe we could phrase it like "not Timelion as a visualization (on dashboards and in the visualize application) or Timelion in Canvas."